### PR TITLE
Added phpcs to CI

### PR DESCRIPTION
**Pull request by bkuhl**
@millidavids @jlaswell

### CODE REVIEW

File: src/BuilderServiceProvider.php [hunk 1]
```diff
@@ -2,7 +2,9 @@
 namespace RealPage\Builder;
 
 use Illuminate\Support\ServiceProvider;
-use RealPage\Builder\Console\{InstallAllDependencies, InstallComposerDependencies, InstallNpmDependencies};
+use RealPage\Builder\Console\InstallAllDependencies;
+use RealPage\Builder\Console\InstallComposerDependencies;
+use RealPage\Builder\Console\InstallNpmDependencies;
```
Comment (bkuhl) [inline src/BuilderServiceProvider.php]: Support for grouped imports is coming soon: https://github.com/squizlabs/PHP_CodeSniffer/issues/878

Comment (jlaswell) [inline src/BuilderServiceProvider.php]: :+1:

**Comment by jlaswell** [comment]
We should probably run phpcs before phpunit in case the sniffer breaks anything.

**Comment by bkuhl** [comment]
Done - I'll consider that an approval

**Pull request merged**